src/main/java/com/paypal/security/SecurityConfiguration.java

```diff
@@ -101,6 +101,10 @@ public int getLdapConnectionPoolMaxSize() {
     return Integer.parseInt(properties.getProperty("ldap.connection.pool.max.size"));
   }
 
+  public int getSuggestionsReloadSleepTime() {
+    return Integer.parseInt(properties.getProperty("suggestions.reload.sleep.time"));
```

Comment:
I've set a convention that anything that is not strictly security enabling parameters carry "nna." prepends.
I think we should do the same here -> "nna.suggestions.reload.sleep.time"

Preferably it should be -> "nna.suggestions.reload.sleep.ms"